### ISSUE TITLE
怎么理解DualTaskLoss？

### ISSUE THREAD

**Issue by lpengc** [yoooooohu/my-net#1]
loss_ewise = F.l1_loss(g, g_hat, reduction='none', reduce=False) 
        p_plus_g_mask = (g >= th).detach().float()
        loss_p_plus_g = torch.sum(loss_ewise * p_plus_g_mask) / (torch.sum(p_plus_g_mask) + eps)

        p_plus_g_hat_mask = (g_hat >= th).detach().float()
        loss_p_plus_g_hat = torch.sum(loss_ewise * p_plus_g_hat_mask) / (torch.sum(p_plus_g_hat_mask) + eps)

        total_loss = 0.5 * loss_p_plus_g + 0.5 * loss_p_plus_g_hat

我理解的是loss_ewise是想要预测结果与标签的边缘差异要小，但是loss_p_plus_g 和loss_p_plus_g_hat是什么意思不太明白，可以帮忙解释一下吗~太感谢啦
